### DISCUSSION TITLE
Difference between clone/lfs clone?

### DISCUSSION THREAD

**Discussion by n2uts** [git-lfs/git-lfs#5531] [Q&A] [answered]
In the [documentation](https://github.com/git-lfs/git-lfs/blob/main/docs/man/git-lfs-clone.adoc), it says that git lfs clone is faster.However, it appears that this flag has long been deprecated. https://github.com/git-lfs/git-lfs/pull/2526

When I try to compare the differences (e.g.)

```bash
$git clone https://huggingface.co/stabilityai/sdxl-vae
Cloning into 'sdxl-vae'...
remote: Total 28 (delta 10), reused 0 (delta 0), pack-reused 2
Unpacking objects: 100% (28/28), 4.69 KiB | 218.00 KiB/s, done.
# display nothing (quiet) and a few moments later 
Filtering content: 100% (3/3), 957.49 MiB | 9.69 MiB/s, done.

$git-lfs clone https://huggingface.co/stabilityai/sdxl-vae
Cloning into 'sdxl-vae'...
remote: Total 28 (delta 0), reused 0 (delta 0), pack-reused 28
Unpacking objects: 100% (28/28), 4.77 KiB | 305.00 KiB/s, done.
# real-time display of download speed & progress
Downloading LFS objects:   0% (0/3), 25 MB | 3.9 MB/s
```
`git-lfs clone` will inform the download progress, Is there an equivalent flag for `git clone` to do the same?  `--verbose` or `--progress` plays no role here. Any help would be much appreciated.

**Comment by bk2204** [comment, marked as answer]
Hey,

In general, unless you need the `-I` or `-X` options, you should use `git clone`.  The two have equivalent speed nowadays in almost all cases.

The reason that there's no option for `git clone` to see the progress of downloading LFS objects is because to Git, Git LFS is a filter process, and so it says, “Filtering content,” and Git LFS would be fighting for Git over the text printed to the terminal, causing a mess.  The “Filtering content” message appears after a few seconds because Git doesn't want to print that message unnecessarily, but it will provide progress if, after a few seconds, the Git LFS operation is not complete.  It will actually show progress if the operation takes more than a few seconds.
